### PR TITLE
fix implementation for ListNotes

### PR DESCRIPTION
**Pull request by mrparkers**
I didn't want to write integration tests for this without `BatchCreateNotes` being implemented, so I backfilled other integration tests instead.

### CODE REVIEW

File: test/v1beta1/note_test.go [hunk 1]
```diff
@@ -1,7 +1,54 @@
...
 func TestNote(t *testing.T) {
-	t.Skipf("TODO")
+	Expect := util.NewExpect(t)
+	s := util.NewSetup()
+
+	// setup project for occurrences
+	projectName := util.RandomProjectName()
+
+	// generate note ID (note IDs are provided by client, not generated by server)
+	noteId := fake.UUID()
```
Comment (rcoy-v) [inline test/v1beta1/note_test.go:21]: This seems like something that should be different for every test case right? Not sure if should be defined once top-level.

File: test/v1beta1/occurrence_test.go [hunk 2]
```diff
@@ -128,6 +77,119 @@ func TestOccurrence(t *testing.T) {
...
+		occurrences[0].Resource.Uri = occurrences[1].Resource.Uri
+
+		// create
+		_, err = s.Gc.BatchCreateOccurrences(s.Ctx, &grafeas_go_proto.BatchCreateOccurrencesRequest{
+			Parent:      listProjectName,
+			Occurrences: occurrences,
+		})
+		Expect(err).ToNot(HaveOccurred())
+
+		t.Run("should be successful", func(t *testing.T) {
```
Comment (rcoy-v) [inline test/v1beta1/occurrence_test.go:103]: I think the listing occurrences and filtering test could be done similar to projects test https://github.com/liatrio/grafeas-elasticsearch/blob/main/test/v1beta1/project_test.go#L26.

You can still have crafted setup more matches or not, and iterate of filters and expected output. Makes it easier if we add more filtering logic in the future.

File: test/v1beta1/note_test.go (hunk 1 above)
Comment (mrparkers) [reply test/v1beta1/note_test.go:21]: oops, I was writing this as if it was in a `BeforeEach`

**Review by ChrisSchreiber — APPROVED** [review]
Looks good

File: test/v1beta1/occurrence_test.go [hunk 3]
```diff
@@ -12,6 +15,12 @@ import (
 	"testing"
 )
 
+type occurrenceFilterTestCase struct {
```
Comment (rcoy-v) [inline test/v1beta1/occurrence_test.go:18]: You could inline this, or declare within function scope to not have as specific of a name.

**Pull request merged**